src/main/resources/com/google/api/codegen/ruby/test.snip

```diff
@@ -0,0 +1,124 @@
+@extends "ruby/method_sample.snip"
+
+@snippet generate(testViewModel)
+  # TODO: Use testViewModel.fileHeader here and in main.snip.
+  @# Copyright 2016 Google Inc. All rights reserved.
+  @#
+  @# Licensed under the Apache License, Version 2.0 (the "License");
+  @# you may not use this file except in compliance with the License.
+  @# You may obtain a copy of the License at
+  @#
+  @#     http://www.apache.org/licenses/LICENSE-2.0
+  @#
+  @# Unless required by applicable law or agreed to in writing, software
+  @# distributed under the License is distributed on an "AS IS" BASIS,
+  @# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  @# See the License for the specific language governing permissions and
+  @# limitations under the License.
+
+  require "minitest/autorun"
+  require "{@testViewModel.libraryUnderTest.outputFile}"
+  include {@testViewModel.libraryUnderTest.namespace}
```

Comment:
I was trying to avoid what the current samples do of having a big block of aliasing each class. I presumed the `include` would be how people use the generated library, so it made sense to do the same in the test.

IIRC, I got errors when trying to scope the include other than globally.

I hadn't thought of aliasing the namespace itself. As it's just one line, I'd be all on for it.